docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -143,6 +143,9 @@ Key interfaces
 
     .. versionadded:: 0.9
 
+    DH private key objects have methods from this interface as well as
+    :class:`DHPrivateKeyWithSerialization`.
```

Comment:
That phrasing sounds good; you can also try `Many DH private keys also implement [...]`?